augur/config.py

```diff
@@ -33,6 +33,34 @@
                     "model": "issues",
                     "repo_group_id": 0
                 },
+                {
+                    "all_focused": 1,
+                    "delay": 150000,
+                    "given": [
+                        "git_url"
+                    ],
+                    "model": "merge_requests",
```

Comment:
Is this **GitLab** merge requests, or just merge requests of any kind? Context clues tell me it's just for GitLab, so I think we need to be very specific both in the model and in the given. Merge request is of course a very common FOSS term and could create confusion if we do no use it carefully.

Reply:
This is *GitLab*

Reply:
Okay, then I think we should change the model to `gitlab_merge_requests` and the given to `gitlab_url` to be as clear as possible.

Reply:
I don't think is resolved, I still think this needs to be changed because it is confusing otherwise. Or at least, I haven't been convinced otherwise yet. 😉

Reply:
After further thought this doesn't need to block this PR, as we should do the same thing for GitHub. We can do that later. But I still think it needs to be changed.